packages/http-client-java/emitter/src/type-utils.ts

```diff
@@ -217,8 +218,9 @@ export function getUnionDescription(union: Union, typeNameOptions: TypeNameOptio
   return name;
 }
 
-export function modelIs(model: Model, name: string, namespace: string): boolean {
-  let currentModel: Model | undefined = model;
+export function modelIs(model: SdkModelType | undefined, name: string, namespace: string): boolean {
```

Comment:
I don't think `model=undefined` case should be handled in this func.

It should be handled before getting into this.